config/example.yaml

```diff
@@ -5,17 +5,18 @@ receivers:
   # the agent is running on every minute.
   # For more information on configuring the hostmetrics receiver, refer to the documentation here:
   # https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver
-  hostmetrics:
-    collection_interval: 1m
-    scrapers:
-      # Theses scrapers work on all operating systems
-      load:
-      filesystem:
-      memory:
-      network:
-      # Remove these scrapers for macOS agents as they are not supported and will generate errors in the logs
-      cpu:
-      disk:
+  # hostmetrics:
+  #   collection_interval: 1m
+  #   scrapers:
+  #     # Theses scrapers work on all operating systems
+  #     load:
+  #     filesystem:
+  #     memory:
+  #     network:
+  #     # Remove these scrapers for macOS agents as they are not supported and will generate errors in the logs
+  #     cpu:
+  #     disk:
```

Comment:
I think we could just remove the commented components, maybe add a comment with doc links?

Reply:
Possibly, but I liked leaving them there for examples. Which we already did for logs in previous version of the script.